indra/llui/llcombobox.cpp

```diff
@@ -256,6 +251,11 @@ bool LLComboBox::itemExists(const std::string& name)
     return mList->getItemByLabel(name);
 }
 
+std::vector<LLScrollListItem*> LLComboBox::getAllData() const
+{
+    return mList->getAllData();
```

Comment:
Good question.  I performed an audit and learned:

`LLComboBox::mList` is set in the **ctor** via `LLUICtrlFactory::create<LLScrollListCtrl>(params)` which eventually invokes `LLUICtrlFactory::createWidgetImpl<LLScrollListCtrl>(params)` which never returns `nullptr`.  On failure it provides an empty list.

Meanwhile, there are a few places in `LLComboBox` where we check for null `mList`.  Those checks should probably be removed since it will never be null and it misleads future readers by suggesting it could be.